File: java/arrayChange.java
int arrayChange(int[] inputArray) {
    //Arrays.sort(inputArray);
    int iteraciones = 0;
    for(int x = 0; x<inputArray.length-1;x++){
        if(inputArray[x]<inputArray[x+1]){
            iteraciones+=0;
        } else{
            while(inputArray[x+1]<=inputArray[x]) {
                inputArray[x + 1]++;
                iteraciones++;
            }
        }
    }
    return iteraciones;
}

File: arrayChange.py
def arrayChange(inputArray):
    c=0
    for i in range(1,len(inputArray)):    
        if inputArray[i-1]>=inputArray[i]:
            c+=((inputArray[i-1]+1)-inputArray[i])
            inputArray[i]=inputArray[i-1]+1                    
    return c